docs/Running-behind-nginx.md

```diff
@@ -114,25 +152,25 @@ Of course you can add as many backend servers as you like.
 
 Using the above, you access Netdata on the backend servers, like this:
 
-- `http://nginx.server/netdata/server1/` to reach `backend-server1`
-- `http://nginx.server/netdata/server2/` to reach `backend-server2`
+- `http://netdata.example.com/netdata/server1/` to reach `backend-server1`
+- `http://netdata.example.com/netdata/server2/` to reach `backend-server2`
 
 ### Using TLS communication
 
-In case Netdata's web server has been [configured to use TLS](../web/server/#enabling-tls-support), you must also encrypt the communication between Nginx and Netdata.
-
-To enable encryption, first [enable TLS/SSL on Nginx](http://nginx.org/en/docs/http/configuring_https_servers.html) and then put the following in the location section of your Nginx configuration:
+In case Netdata's web server has been [configured to use TLS](../web/server/#enabling-tls-support), it is necessary to specify inside the Nginx configuration that the final destination is using TLS. To do this, please, append the following parameters in your `nginx.conf`
 
 ```
 proxy_set_header X-Forwarded-Proto https;
 proxy_pass https://localhost:19999;
 ```
 
-If nginx is not configured as described here, you will probably receive the error `SSL_ERROR_RX_RECORD_TOO_LONG`.
+Optionally it is also possible to [enable TLS/SSL on Nginx](http://nginx.org/en/docs/http/configuring_https_servers.html), this way the user will encrypt not only the communication between Nginx and Netdata but also between the user and Nginx.
+
+If Nginx is not configured as described here, you will probably receive the error `SSL_ERROR_RX_RECORD_TOO_LONG`.
 
 ### Enable authentication
 
-Create an authentication file to enable basic authentication via Nginx. Do not use authentication without having first [enabled TLS](#using-tls-communication)!
+Create an authentication file to enable basic authentication via Nginx, this secures your Netdata dashboard. Do not use authentication without having first [enabled TLS](#using-tls-communication)!
```

Comment:
Please remove the "Do not use authentication without..." completely. 
Perhaps Thiago's comment in the last PR was confusing, but this statement is completely wrong and we didn't catch it.

Reply:
I agree with @cakrit , let me try to explain better. When we have the Netdata set with TLS, it is a very good idea to use it, this is the motive we adjusted the text between the lines 160-167.
When we "enable authentication" in the Nginx, it does not have direct relationship with the "using-tls-communicaton", it is completely possible to have the TLS without to use Nginx to authenticate and the opposite is also true, so please remove "Do not use authentication without having first [enabled TLS](#using-tls-communication)!"

Reply:
It was a bit confusing to me as well.
I have effected the change

Reply:
Let me explain better the situation. When we activate the TLS on Netdata, this means that connections direct with Netdata will use encrypt channel, but the user is free to allow connections between him and Nginx without encryptation. In the same way, it is possible the user thinks that it is not necessary to encrypt the connection between Nginx and netdata,  because he created an authentication in his Nginx, so for I person access the Netdata, it will be necessary to authenticate firstly in Nginx. Finally, an user that wanna have everything encrypted, can activate TLS on Netdata and also set Auhtentication in Nginx with TLS. All the scenarios are possible, our goal is only to say for the user that they are possible, but there is not any obligation to activate two channels together or three, the user is free to decide.

Reply:
I understand now.

The goal is for the user to have freedom of choice from different possibilities.

Reply:
Exactly!